scripts/build-ios.sh

```diff
@@ -1,10 +1,10 @@
 #!/bin/bash
 
-cordova build ios
-mkdir -p platforms/ios/build/appium/Payload
-cp -R platforms/ios/build/emulator/Ionic\ Conference\ App.app platforms/ios/build/appium/Payload
+ionic cordova build ios
+# mkdir -p platforms/ios/build/appium/Payload
```

Comment:
These steps are not needed for iOS apps for sims. The `{app-name}.app` can also be used by Appium to install the app